src/interface/meta.thrift

```diff
@@ -321,6 +327,8 @@ service MetaService {
     ExecResp remove(1: RemoveReq req);
     ExecResp removeRange(1: RemoveRangeReq req);
     ScanResp scan(1: ScanReq req);
+    KeyOrValueScanResp scanKey(1: ScanReq req);
```

Comment:
Do we really need the two interfaces?

Reply:
Actually, I considered use a flag to distinguish scan the keys and the values.

But I find it‘s confuse.

Reply:
I mean do we really need to scan key or value only? 
There is no efficient problems for the interfaces in meta.